OmniKit/PumpManager/OmnipodPumpManager.swift

```diff
@@ -1038,11 +1038,19 @@ extension OmnipodPumpManager {
             completion(String(describing: OmnipodPumpManagerError.noPodPaired))
             return
         }
+#if SKIP_ACTIVATION_TIME_EXCEEDED_CHECKING
```

Comment:
Please do not use conditional compilation blocks.  There are a lot of things listed in the summary. It would be nice if this was broken up.